client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -138,8 +148,31 @@ class ContentDataProvider
     dataTransfer.set(this.dragMimeTypes[0], dataTransferItem);
   }
 
+  public async provideDocumentDropEdits(
+    _document: TextDocument,
+    position: Position,
+    dataTransfer: DataTransfer,
+    token: CancellationToken,
+  ): Promise<DocumentDropEdit | undefined> {
+    const dataTransferItem = dataTransfer.get(this.dragMimeTypes[0]);
+    const contentItem =
+      dataTransferItem && JSON.parse(dataTransferItem.value)[0];
+    if (token.isCancellationRequested || !contentItem) {
+      return undefined;
+    }
+
+    const fileFolderPath = await this.model.getFileFolderPath(contentItem);
+    if (!fileFolderPath) {
+      return undefined;
+    }
+
+    return {
+      insertText: `filename myfile filesrvc folderpath='${fileFolderPath}' filename='${contentItem.name}';\n`,
```

Comment:
nit: just put a note here that we have customers who like uppercase as #22
I don't know is it a good idea to insert uppercase if the word before or after the position is uppercase?
I'll be OK with simply moving forward and revisit if customer reports.

Reply:
oh, another thing is that is `myfile` good enough? So if user drop 2 files they have to change the name. Or how about simply using `${contentItem.name}`?

Reply:
Good question. I originally copied from the issue w/o _really_ thinking this part (the name) through. Will take a look. As for uppercase or lowercase, I'm inclined to keep it as-is because it seems like the simplest approach. But yeah, if a customer reports it, lets adjust :)

Reply:
Addressed in https://github.com/sassoftware/vscode-sas-extension/pull/510/commits/41a228c36f53edb8f8cdd803f1a7412e3451e9a8